src/core/file.js

```diff
@@ -36,6 +37,19 @@ module.exports = {
     });
   },
 
+  move(filePath, targetDir, extraFile, logger = defaultLogger) {
+    logger.info(`Moving the ${filePath} into ${targetDir}.`);
+    const target = path.join(targetDir, extraFile.path);
+    // Only move file with override status
+    if (extraFile.config === 'override') {
```

Comment:
what happen if extraFile.config is skipIFExist

Reply:
I think we just consider the case that extraFile.config is override. 
In case skipIFExist, I don't move the file to tmp directory.

Reply:
I think you misunderstood
The ignoreIfExist configuration means

If the file already exists, we don't replace it with the newly-downloaded file
Otherwise, we use the existing file

Reply:
@lequangtrai , that means: in case skipIfExist, we don't perform file downloading if the file already exists in KS project. Otherwise, download file and replace as usual.

Reply:
As the discussion, I updated the logic to download file if the file is not exists in KS project.